modules/jaeger-install.adoc

```diff
@@ -0,0 +1,48 @@
+////
+This PROCEDURE module included in the following assemblies:
+- rhbjaeger-installation.adoc
+////
+
+[id="jaeger-operator-install_{context}"]
+= Installing the Jaeger Operator
+
+To install Jaeger you use the link:https://operatorhub.io/[OperatorHub] to install the Jaeger Operator.
+
+By default the Operator is installed in the `openshift-operators` project.
+
+.Prerequisites
+* Access to the {product-title} web console.
+* An account with the `cluster-admin` role.
+* If you require persistent storage, you must also install the Elasticsearch Operator before installing the Jaeger Operator.
+
+[WARNING]
+====
+Do not install Community versions of the Operators. Community Operators are not supported.
+====
+
+.Procedure
+
+. Log in to the {product-title} web console as a user with the `cluster-admin` role.
+
+. Navigate to *Operators* -> *OperatorHub*.
+
+. Type *Jaeger* into the filter box to locate the Jaeger Operator.
+
+. Click the *Jaeger Operator* provided by Red Hat to display information about the Operator.
+
+. Click *Install*.
+
+. On the *Create Operator Subscription* page, select *All namespaces on the cluster (default)*. This installs the Operator in the default `openshift-operators` project and makes the Operator available to all projects in the cluster.
```

Comment:
Should this para mention installing in the `openshift-operators` namespace/project, if at the bottom of the doc it mentions selecting the `openshift-operators` project (as if the user has a choice)?

Reply:
It installs into openshift-operators by default (so you don't have a choice).  But the UI continues to display the "default" namespace, so if you don't select the openshift-operators namespace you cannot see the installation.